File: Java/src/main/java/com/techreturners/Parrot.java
package com.techreturners;

public class Parrot extends Bird{
    public Parrot(String name, String colour){
        super(name, colour);
    }

    @Override
    public void speak() {
        System.out.println("Hello! I'm " + name + " and I'm a " + color + " parrot.");
        System.out.println("Happy Bird-day to meeeee!!!");
    }
}

File: CSharp/Parrot.cs
using System;

public class Parrot : Bird
{
	public Parrot(string name, string color) : base(name, color)
	{
	}

    public override void Speak()
    {
        Console.WriteLine($"Hi! I'm {Name} and I'm a {Color} parrot.");
        Console.WriteLine("Happy Bird-day to meee!");
    }
}
